seqr/views/utils/dataset_utils.py

```diff
@@ -320,6 +321,38 @@ def _parse_tsv_row(row):
 INDIV_ID_COL = 'individual_id'
 TPM_HEADER_COLS = [SAMPLE_ID_COL, PROJECT_COL, GENE_ID_COL, TISSUE_COL, TPM_COL]
 
+CHROM_COL = 'chrom'
+START_COL = 'start'
+END_COL = 'end'
+STRAND_COL = 'strand'
+READ_COUNT_COL = 'read_count'
+DOT_SIZE_COL = 'dot_size'
+SPLICE_TYPE_COL = 'type'
+P_VALUE_COL ='p_value'
```

Comment:
wouldn't the be shared with the existing outlier columns?

Reply:
Splice Junction Outlier and existing Outlier have the `p_value` and `z_score` columns. But the Outlier has a `p_adjust` column which Splice Junction Outlier doesn't need. Otherwise, using the `Outlier` model as a base class for the `Splice Junction Outlier` model might be good.

Reply:
lets keep them separate, I think they are different enough that the savings we get from sharing the model aren't worth it